src/Core/Framework/Bundle.php

```diff
@@ -100,20 +105,27 @@ public function configureRouteOverwrites(RouteCollectionBuilder $routes, string
         }
     }
 
-    /**
-     * @feature-deprecated tag:v6.4.0 - Implement `getActionEventClasses` instead
```

Comment:
why did you remove the deprecation?

Reply:
I'm not sure why this was removed. I just added it again